jitify2.hpp

```diff
@@ -5844,7 +5844,19 @@ class ProgramCache {
           [&](std::istream& istream) {
             return LinkedProgram::deserialize(istream);
           });
-      if (!error.empty()) return LoadedProgram::Error(error);
+
+      if (!error.empty())
```

Comment:
Should we check for a specific error here instead of just any error?

Reply:
I don't think we gain much from inspecting the specific error as it exists now. All of them mean "we cant load or save the file cache". Some of which might (?) be nice to report as a warning, but it could also just be noise. Going to work with Ben to figure out a better solution over-all.

Reply:
The down side is if file caching weren't working _at all_. But I think this code is more in line with what we were doing in cudf jit file caching.